src/platform/user/profile/vet360/containers/AddressValidationModal.jsx

```diff
@@ -286,6 +283,7 @@ AddressValidationModal.propTypes = {
   isAddressValidationModalVisible: PropTypes.bool.isRequired,
   addressValidationError: PropTypes.bool.isRequired,
   suggestedAddresses: PropTypes.array.isRequired,
+  confirmedSuggestions: PropTypes.array.isRequired,
```

Comment:
If you know the shape of each `confirmedSuggestion`, you could do something like:
```js
confirmedSuggestions: PropTypes.arrayOf(PropTypes.shape({
  id: PropTypes.string.isRequired,
  ...
})).isRequired,
```
to get even more out of `prop-types`! 🙂